src/rrulestr.ts

```diff
@@ -23,7 +24,15 @@ type FreqKey = keyof typeof Frequency
 export default class RRuleStr {
   // tslint:disable-next-line:variable-name
   private _handle_DTSTART (rrkwargs: Options, _: any, value: string, __: any) {
+    console.log('HANDLE DTSTART')
```

Comment:
Intentional?

Reply:
I was trying to determine whether this is dead code, and it looks like it indeed is. I intend to do some cleanup in a followup PR since there is considerable duplication (and inconsistency) between `parseString` and `rrulestr`.